rskj-core/src/test/java/org/ethereum/core/LogInfoTest.java

```diff
@@ -19,16 +19,14 @@
 
 package org.ethereum.core;
 
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.util.ByteUtil;
 import org.ethereum.vm.LogInfo;
-
+import org.junit.Ignore;
```

Comment:
Are ignored tests?

Reply:
yes, correct. One test is ignored in this class.